### ISSUE TITLE
apm develop doesn't work on apm

### ISSUE THREAD

**Issue by mnquintana** [atom/apm#257]
`apm develop apm` returns `Request for package information failed: Not Found`
`apm develop <any-other-core-package>` works normally

```
apm  0.119.0
npm  1.4.4
node 0.10.35
python 2.7.6
git 2.2.2
```

**Comment by kevinsawicki** [comment]
Yeah, this is expected, `apm` is not a package published to atom.io, it is a library published to npmjs.org

We could special-case this though to make it easier to hack on apm.

**Comment by mnquintana** [comment]
Ah okay, I had a feeling it was something like that.

I think making it a special case seems like a good idea - if it turns out that isn't the best route, I think the best compromise would be to display a warning and help text for the preferred way to hack on apm.

**Comment by kevinsawicki** [comment]
:+1:
